sensor_msgs/Image image
string class_name
---
bool success